tests/test_examples.py

```diff
@@ -877,7 +877,7 @@ def mirrorRatio(params):
     )
     optimizer = Optimizer("lsq-auglag")
     (eq, field), _ = optimizer.optimize(
-        (eq, field), objective, constraints, maxiter=100, verbose=3
+        (eq, field), objective, constraints, maxiter=150, verbose=3
```

Comment:
Question: Is this necessary for the test to pass or to see If it terminates properly? Why would changing the stopping criterion change the end result after 100 steps? I mean it doesn't change the step itself, right?

Reply:
for the augmented lagrangian methods this also affects the internal criteria to determine when to update the multipliers and penalty param, so it can affect the results somewhat. The old test wasn't actually terminating on a tolerance, it was just running to maxiter, now it correctly terminates when it reaches an actual tolerance